packages/devtools_app/lib/src/flutter/custom_pointer_scroll_view.dart

```diff
@@ -0,0 +1,874 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:math' as math;
+import 'dart:ui';
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/scheduler.dart';
+import 'package:flutter/widgets.dart';
+
```

Comment:
Majority of code in this file is copied from Flutter. I will call out changes with comments.